### PR TITLE
Announce release policy

### PR DESCRIPTION
**Pull request by waiting-for-dev**
Blog post to announce our new [release policy](https://solidus.io/release_policy).

Creating a draft, as I need to ask @mfrecchiami for a picture, but you can already review the content.

### CODE REVIEW

**Comment by waiting-for-dev** [comment]
I added the image. This is ready for review 🙂

File: source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Nonetheless, we reckon we haven't been consistent on the timing for those
+releases. That had made some folks wonder when that loved new feature or when
+the latest Rails version would be officially integrated. Definitely, that's
+something that doesn't help to plan upgrades.
+
+We want to improve there. Because of that, we discussed and agreed on a release
+policy that we hope will help the community prepare for Solidus upgrades. As
+always in Open Source, resources are limited, but we're confident that this
+self-inflicted pressure will force us to keep pace with your expectations.
```
Comment (elia) [inline source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown:23]: thoughts on `force us` → `help us`?

File: source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Please, read the [whole document on our
+site](https://solidus.io/release_policy/), but if you're tight on time, here
+are the principal takeaways:
+
+- A quarterly new minor release.
+- Major releases should not be more than two years apart.
+- Patch releases at every backported fix.
+
+Share your ideas. Please, don't think twice about reaching us on our [Slack
```
Comment (elia) [inline source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown:33]: not sure but worth checking, `reaching us` → `reaching out to us`

**Review by elia — APPROVED** [review]
👍 just left a couple of optional nits

File: source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown (hunk 1 above)
Comment (waiting-for-dev) [reply source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown]: I like it. Less aggressive. Done!

File: source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown (hunk 2 above)
Comment (waiting-for-dev) [reply source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown]: I think you're right. Done!

File: source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+workspace](http://slack.solidus.io/), and let us know any suggestions or
+concerns you might experience. We're here to make your experience with the most
```
[the remark this answers is not in the document]
Comment (waiting-for-dev) [reply source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown]: Yeah. Also done :slightly_smiling_face:

**Comment by waiting-for-dev** [comment]
> +1 just left a couple of optional nits

Thanks for your suggestions, @elia. I incorporated all of them. I also updated the publication date to today.

**Review by kennyadsl — APPROVED** [review]

**Pull request merged**